src/llvmir2hll/graphs/cfg/cfg_traversal.cpp

```diff
@@ -168,7 +168,7 @@ bool CFGTraversal::getCurrRetVal() const {
 */
```

Comment:
The descriptions of the two modified functions are
```
* @brief A recursive implementation of performTraversal().
```
```
* @brief A recursive implementation of performReverseTraversal().
```
I guess we should remove the word _recursive_ from them as they are no longer recursive.

Reply:
Changed to non-recursive